docs/desktop-wpf/fundamentals/styles-templates-overview.md

```diff
@@ -118,6 +167,48 @@ In the following illustration, the mouse is pointing to the third item.
 
 In addition to <xref:System.Windows.Trigger> and <xref:System.Windows.EventTrigger>, there are other types of triggers. <xref:System.Windows.MultiTrigger> allows you to set property values based on multiple conditions. You use <xref:System.Windows.DataTrigger> and <xref:System.Windows.MultiDataTrigger> when the property of your condition is data-bound.
 
+## Visual States
```

Comment:
The C#/VB code samples on line 121 are both showing up, no matter which language you select in the selector.